### PR TITLE
Fixes KEYCLOAK-7384 - Linked Accounts HTML Update

### PR DESCRIPTION
**Pull request by zhutaoredhat**
This PR is to provide HTML and CSS code 
JIRA task: https://issues.jboss.org/browse/KEYCLOAK-7384

The HiDPI Screen (1440px) :
pc

On mobile
mobile

@ssilvert Please have a review~

### CODE REVIEW

File: themes/src/main/resources/theme/keycloak-preview/account/resources/styles.css [hunk 1]
```diff
@@ -301,11 +301,35 @@ p.description {
   position: absolute;
   right: 10px;
 }
+.list-view-pf-additional-info .fa-user{
```
Comment (junezhang) [inline themes/src/main/resources/theme/keycloak-preview/account/resources/styles.css]: need to add space between the selector and curly braces

File: themes/src/main/resources/theme/keycloak-preview/account/resources/styles.css [hunk 2]
```diff
@@ -301,11 +301,35 @@ p.description {
...
+  margin-right: 5px;
+}
+.linked-account-list .list-view-pf .list-group-item-heading {
+    -ms-flex: 1 0 calc(40% - 20px);
```
Comment (junezhang) [inline themes/src/main/resources/theme/keycloak-preview/account/resources/styles.css]: no need to add -ms-flex.
PatternFly uses Autoprefixer to deal with CSS vendor prefixes.
Do not add vender prefixes to your code, Autoprefixer will do it for you on build.

File: themes/src/main/resources/theme/keycloak-preview/account/resources/styles.css [hunk 3]
```diff
@@ -301,11 +301,35 @@ p.description {
...
+    flex: 1 0 calc(40% - 20px);
+  }
+.linked-account-list .list-view-pf-additional-info{
+    width:10%
```
Comment (junezhang) [inline themes/src/main/resources/theme/keycloak-preview/account/resources/styles.css]: 1, need to add space between the selector and curly braces
2, Include one space after : for each declaration.
3, add ";" end of declaration

File: themes/src/main/resources/theme/keycloak-preview/account/resources/styles.css [hunk 4]
```diff
@@ -301,11 +301,35 @@ p.description {
...
+.linked-account-list .list-view-pf .list-group-item-text {
+    width: calc(45% - 40px);
+  }
 
 @media (max-width: 767px) {
   .card-pf-application h2 {
     margin-top: -10px;
   }
+  .card-linked-account .card-pf-body{
+    padding-left: 0;
```
Comment (junezhang) [inline themes/src/main/resources/theme/keycloak-preview/account/resources/styles.css]: need to add space between the selector and curly braces

File: themes/src/main/resources/theme/keycloak-preview/account/resources/styles.css [hunk 5]
```diff
@@ -301,11 +301,35 @@ p.description {
...
+    padding-right: 0;
+  }
+  .card-linked-account .list-group-item{
+    padding: 10px 0px;
```
Comment (junezhang) [inline themes/src/main/resources/theme/keycloak-preview/account/resources/styles.css]: 1, need to add space between the selector and curly braces
2, remove the "px", avoid specifying units for zero values

File: themes/src/main/resources/theme/keycloak-preview/account/resources/app/content/linked-accounts-page/linked-accounts-page.component.html [hunk 6]
```diff
@@ -1 +1,126 @@
...
+  <div class="card-pf-body row">
+    <div class="col-md-3">
+      <div class="card-pf-subtitle">
+        Authorized Provider
+      </div>
+      <div class="introMessage">
+        <p>Authorized Providers linked with your account.</p>
+      </div>
+    </div>
+    <div class="col-md-9 linked-account-list">
```
Comment (junezhang) [inline themes/src/main/resources/theme/keycloak-preview/account/resources/app/content/linked-accounts-page/linked-accounts-page.component.html]: Do we need to add the second new css "linked-account-list", as above I found you already added "card-linked-account"?

**Review by junezhang — COMMENTED** [review]
Overview one thing is for the new class name, do we need to add two new class?
the second thing is some code-style issues.

**Comment by zhutaoredhat** [comment]
This PR is to provide HTML and CSS code 
JIRA task: https://issues.jboss.org/browse/KEYCLOAK-7384

The HiDPI Screen (1440px) :
pc

On mobile
mobile

@stianst Please have a review~

**Review by junezhang — DISMISSED** [review]

**Pull request closed without merging**